src/pages/AddData.tsx

```diff
@@ -32,36 +50,20 @@ const BulkUploaderLoader = () => {
       fileName: file.name,
       sourceKey: presignedPost.fields.key,
     });
+    onBulkUpload();
```

Comment:
Mind throwing an extra newline above this (for consistency with the rest of the function)?

Reply:
The 👎🏻 is meant as "no, I don't mind!"

Reply:
@slifty Dan I just burstlolled at how chaotic that was.